src/createLoggerMiddleware.js

```diff
@@ -32,9 +32,28 @@ export default ({logger}) => {
         const meta = {name: 'express'};
 
         if (req.method === 'POST' && req.body && req.body.operationName) {
+          let variables;
+
+          if (MAX_GRAPHQL_VARIABLES_LOG_LENGTH > 0) {
```

Comment:
A value of `0` could be reserved for disabling logging of variables, maybe adjust this to `>= 0`?

This should be mentioned in the README too.